schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Thunderbird In-App Notifications",
+  "type": "array",
+  "items": [
+    {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "format": "uuid",
+          "description": "Unique ID set by the server"
+        },
+        "start_at": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Timestamp after which Thunderbird will show the event after startup."
+        },
+        "end_at": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Timestamp after which Thunderbird will never show the event, even if it has never been shown to the user."
+        },
+        "title": {
+          "type": "string",
+          "description": "Short sentence describing the event which will be displayed in the Thunderbird UI."
+        },
+        "description": {
+          "type": "string",
+          "description": "A short paragraph that can contain HTML and will be displayed in the Thunderbird UI."
```

Comment:
I think we should limit the allowed HTML so we can sanitize it and display it confidently. Obviously we can do this unilaterally in the client anyway, but maybe we'd want to define that somewhere.